src/main/java/com/airlines/dao/CrewMemberDaoImpl.java

```diff
@@ -0,0 +1,145 @@
+
+    }
+
+    private PreparedStatement prepareInsertStatement(Connection connection, CrewMember crewMember) {
+        try {
+            PreparedStatement insertStatement = connection.prepareStatement(INSERT_CREWMEMBER_SQL, PreparedStatement.RETURN_GENERATED_KEYS);
+            return fillStatementWithAccountData(insertStatement, crewMember);
+        } catch (SQLException e) {
+            throw new DaoOperationException("Cannot prepare statement to insert account", e);
+        }
+    }
+
+    private PreparedStatement fillStatementWithAccountData(PreparedStatement insertStatement, CrewMember crewMember)
+            throws SQLException {
+        insertStatement.setString(1, crewMember.getFirsName());
+        insertStatement.setString(2, crewMember.getLastName());
+        insertStatement.setString(3, String.valueOf(crewMember.getPosition()));
+        insertStatement.setDate(4, Date.valueOf(crewMember.getBirthday()));
+        insertStatement.setString(5, String.valueOf(crewMember.getCitizenship()));
+        return insertStatement;
+    }
+
+    private void executeUpdate(PreparedStatement insertStatement, String errorMessage) throws SQLException {
+        int rowsAffected = insertStatement.executeUpdate();
+        if (rowsAffected == 0) {
+            throw new DaoOperationException(errorMessage);
+        }
+    }
+
+    private int fetchGeneratedId(PreparedStatement insertStatement) throws SQLException {
+        ResultSet generatedKeys = insertStatement.getGeneratedKeys();
+
+        if (generatedKeys.next()) {
+            return generatedKeys.getInt(1);
+        } else {
+            throw new DaoOperationException("Can not obtain an account ID");
+        }
+    }
+
+    public CrewMember findOne(int id) {
+        try (Connection connection = dataSource.getConnection()) {
+            return findCrewMemberById(id, connection);
+        } catch (SQLException e) {
+            throw new DaoOperationException(String.format("Cannot find Account by id = %d", id), e);
+        }
+    }
+
+    private CrewMember findCrewMemberById(int id, Connection connection) throws SQLException {
+        PreparedStatement selectByIdStatement = prepareSelectByIdStatement(id, connection);
+        ResultSet resultSet = selectByIdStatement.executeQuery();
+        resultSet.next();
+        return parseRow(resultSet);
+    }
+
+    private PreparedStatement prepareSelectByIdStatement(int id, Connection connection) {
+        try {
+            PreparedStatement selectByIdStatement = connection.prepareStatement(SELECT_CREWMEMBER_BY_ID_SQL);
+            selectByIdStatement.setLong(1, id);
+            return selectByIdStatement;
+        } catch (SQLException e) {
+            throw new DaoOperationException("Cannot prepare statement to select account by id", e);
+        }
+    }
+
+    private CrewMember parseRow(ResultSet resultSet) throws SQLException {
+        CrewMember crewMember = CrewMember.builder()
```

Comment:
you don'n need this variable, you can write just:
private CrewMember parseRow(ResultSet resultSet) throws SQLException {
        return CrewMember.builder()...